packages/mds-jurisdiction-service/server/handlers/utils.ts

```diff
@@ -14,16 +14,32 @@
     limitations under the License.
  */
 
-import { Timestamp, Jurisdiction } from '@mds-core/mds-types'
+import { Timestamp } from '@mds-core/mds-types'
 import { DeepPartial } from 'typeorm'
-import { validateJurisdiction } from '@mds-core/mds-schema-validators'
+import {
+  ValidateSchema,
+  SchemaBuilder,
+  uuidSchema,
+  stringSchema,
+  timestampSchema
+} from '@mds-core/mds-schema-validators'
 import { v4 as uuid } from 'uuid'
-import { CreateJurisdictionType } from '../../@types'
+import { filterEmptyHelper } from '@mds-core/mds-utils'
+import { CreateJurisdictionType, JurisdictionDomainModel } from '../../@types'
 import { JurisdictionEntity } from '../repository/entities'
 
+const jurisdictionSchema = (max: Timestamp = Date.now()) =>
```

Comment:
I'm wondering if it makes sense to break Joi helpers out into their own folder within the service -- for certain services I imagine we'll have a lot of schemas we move down into the service layer, would help with organization.

Not super necessary for this service, but because we're defining a new design pattern for our packages, might be worth doing.